test/dashboard.go

```diff
@@ -0,0 +1,147 @@
+package test
+
+import (
+	"net/http"	
+	"bytes"
+	"net/http/cookiejar"
+	"fmt"	
+	"io/ioutil"
+	"encoding/json"
+)
+
+var sessionCookie []*http.Cookie
```

Comment:
It is not good practice to use global variables. you can write a wrapper function to return these values if they are used at multiple places

Reply:
Hi Vidhut,
Since we are maintaining same session and have same runid for all testsuites, I am using global variables.